packages/langium-vscode/package.json

```diff
@@ -45,10 +45,10 @@
   "scripts": {
     "prepare": "npm run copy-schema",
     "copy-schema": "cpy ../langium-cli/langium-config-schema.json ./data",
-    "vscode:prepublish": "npm run clean && npm run lint && webpack --mode=production",
+    "vscode:prepublish": "npm run clean && tsc --noEmit && npm run lint && node esbuild.js --minify",
     "clean": "rimraf out",
-    "build": "webpack",
-    "watch": "webpack --watch",
+    "build": "node esbuild.js",
```

Comment:
Please also add `tsc --noEmit` to `build` to ensure that the CI build fails when there are type errors.